src/Microsoft.DotNet.Wpf/src/WpfGfx/codegen/mcg/main/Elements.rsp

```diff
@@ -0,0 +1,19 @@
+-s:main\SimpleGenerator.cs
```

Comment:
I created this file and Resources.rsp by hand because they didn't exist. They were probably missed when adding the code in this repo because *.rsp files are excluded in the gitignore.